src/core/server/saved_objects/service/lib/decorate_es_error.ts

```diff
@@ -35,6 +35,8 @@ const {
   NotFound,
   BadRequest,
 } = legacyElasticsearch.errors;
+const SCRIPT_CONTEXT_DISABLED_REGEX = /(?:cannot execute scripts using \[)([a-z]*)(?:\] context)/;
```

Comment:
We only see this message for `deleteByNamespace` when the "update" script context is disabled. E.g., the error reason is `cannot execute scripts using [update] context`. However, I made the regex generic so it can be reused for other script execution failures.